source/python-best-practise.rst

```diff
@@ -0,0 +1,498 @@
+==============
+Bedste praksis
+==============
+
+Jævnfør ordbogen betyder ordet praksis en virkeliggørelse af noget tænkt eller planlagt. Det er noget, vi udøver eller gør. Bedste praksis er altså den bedste virkeliggørelse af noget, vi ønsker at opnå. Det er den bedste måde at få frem på for at opnå dét, vi ønsker at opnå.
+
+Følgende beskriver, hvordan man får en god start på et nyt Python-projekt. Det centrale spørgsmål er **Hvordan er vejen til at opbygge og vedligeholde en velorganiseret Python-pakke?**. Præsentationen går også gradvist frem således, at pakkens komponenter og hjælpemidler bliver beskrevet i en rækkefølge, som er hensigtsmæssig, når man bygger en pakke op fra bunden.
+
+
+Afgrænsning
+-----------
+
+Konkret forsøger vi her at beskrive den nuværende, bedste praksis for programmeringssproget Python.
+
+Vejledningen kan både bruges til opbygning af genbrugelig kode [biblioteker] og programmer [applikationer, programmel]. Indtil videre skelner denne vejledning ikke mellem bibliotek og applikation, da organisationen af kildekode, dokumentation, etc. i projektet som udgangspunkt kan foregå på samme måde langt hen ad vejen.
+
+Python-udvikling involverer en rækker værktøjer og processer, der ikke har med Python-programmering at gøre. Eksempelvis bruger vi Git til versionsstyring, og med GitHub-platformen har vi en central placering til den autoritative historik for arkiverne. Med GitHubs Actions kan platformen desuden bruges som central udviklingsserver, der automatisk kan kvalitetssikre Python-koden, bygge dokumentation og andet. Her beskriver vi kun, hvordan man bruger værktøjer som disse fra et Python-synspunkt.
+
+Indtil videre er fremgangsmåder til pakke-fremstilling [*en.* build] og distribution udeladt. Da vi i organisationen ikke har noget officielt sted at lægge færdigbyggede pakker, vi kan hente fra. Da vi anvender Python-distributionen MambaForge, er det også en mulighed at bygge en pakke i et format, der kan installeres af ``mamba``. Vi beskriver i stedet en fremgangsmåde, hvor brugeren installerer pakken lokalt med git og ``pip``.
+
+Det er desuden ikke formålet her at gå i dybden med Pythons metodologi, økosystem, hovedorganisation og andre bidragende organisationer.
+
+.. Dette kan komme med i en senere version af denne vejledning.
+
+**Det primære fokus her er de standarder, konventioner og værktøjskombinationer, der virker godt idag for os i SDFE.**
+
+
+Informationskilder
+------------------
+
+Bedste praksis er dels udtrykt ved standarder, som defineres og implementeres i organiserede fællesskaber som The Python Software Foundation, men må også tilpasses de konkrete behov og begrænsninger, som er tilstede i den konkrete situation, man sidder i.
```

Comment:
Jeg tror det her virker, da jeg har defineret linket på en anden side. 
```suggestion
Bedste praksis er dels udtrykt ved standarder, som defineres og implementeres i organiserede fællesskaber som `The Python
Software Foundation`_, men må også tilpasses de konkrete behov og begrænsninger, som er tilstede i den konkrete situation, man sidder i.
```

Reply:
Tak. Jeg havde den in mente som et link, der skulle laves.